velox/functions/sparksql/aggregates/tests/BloomFilterAggAggregateTest.cpp

```diff
@@ -71,14 +71,12 @@ TEST_F(BloomFilterAggAggregateTest, emptyInput) {
   testAggregations(vectors, {}, {"bloom_filter_agg(c0, 5, 64)"}, expected);
 }
 
-TEST_F(BloomFilterAggAggregateTest, nullBloomFilter) {
-  auto vectors = {makeRowVector({makeAllNullFlatVector<int64_t>(2)})};
-  auto expectedFake = {makeRowVector(
-      {makeNullableFlatVector<StringView>({std::nullopt}, VARBINARY())})};
-  VELOX_ASSERT_THROW(
-      testAggregations(
-          vectors, {}, {"bloom_filter_agg(c0, 5, 64)"}, expectedFake),
-      "First argument of bloom_filter_agg cannot be null");
+TEST_F(BloomFilterAggAggregateTest, nullInput) {
+  auto vectors = {makeRowVector(
```

Comment:
It seems like we create 2 columns here: c0 and c1. c0 has no nulls. c1 has only nulls. However, testAggregations only uses c0. Why do we need c1? Does this test actually verify the changes? Would this test fail without the code changes? 

Also, notice that sizes of columns c0 and c1 and different. c0 has 100 rows, while c1 only 1. 

Please, take the time to carefully review the test and make sure it actually verifies the changes and doesn't include unnecessary code.

Reply:
Yes. I have updated the unittest.